src/components/Nav/NavItem.react.js

```diff
@@ -48,30 +63,52 @@ class NavItem extends React.PureComponent<Props, State> {
       to,
       type = "li",
       icon,
-      hasSubNav,
+      hasSubNav: forcedHasSubNav,
       active,
       subItems,
       subItemsObjects,
+      position = "bottom-start",
     }: Props = this.props;
-    const navLink = (typeof children === "string" || value) && (
-      <Nav.Link
-        className={className}
-        to={to}
-        icon={icon}
-        RootComponent={LinkComponent}
-        hasSubNav={hasSubNav}
-        active={active}
-      >
-        {!hasSubNav && typeof children === "string" ? children : value}
-      </Nav.Link>
-    );
+
+    const hasSubNav: boolean =
```

Comment:
[nit] while you are welcome to type this as `boolean` for readability, flow will infer it for you.  you probably already know that but just thought I'd let you know just in case